projects/igniteui-angular/src/lib/simple-combo/simple-combo.component.ts

```diff
@@ -192,10 +192,17 @@ export class IgxSimpleComboComponent extends IgxComboBaseDirective implements Co
      * ```
      */
     public select(item: any): void {
-        if (item !== undefined) {
-            const newSelection = this.selectionService.add_items(this.id, item instanceof Array ? item : [item], true);
-            this.setSelection(newSelection);
+        if (item === undefined) {
```

Comment:
At this point the `select` methods for both combos are very similar. So, it should be safe to extract the logic to a common method.